app/assets/javascripts/course/lesson_plan_items.js

```diff
@@ -0,0 +1,18 @@
+// Place all the behaviors and hooks related to the matching controller here.
+// All this logic will automatically be available in application.js.
+(function() {
+  $(document).on('page:load ready', function (){
+    'use strict';
+    var todaysMilestone = $('.milestone-body-past').last()[0];
```

Comment:
Your selector is not specific enough. Another milestone on another page would trigger this behaviour.